packages/webapp/src/inductions/components/induction-lists/inviter-inductions.tsx

```diff
@@ -0,0 +1,134 @@
+import dayjs from "dayjs";
+import * as relativeTime from "dayjs/plugin/relativeTime";
+
+import { getEndorsementsByInductionId } from "inductions/api";
+import { getInductionStatus } from "inductions/utils";
+import { useFetchedData } from "_app";
+import * as InductionTable from "inductions/components/induction-lists/induction-table";
+import { Endorsement, Induction, InductionStatus } from "../../interfaces";
+import { InductionActionButton } from "./action-button";
+
+dayjs.extend(relativeTime.default);
+
+interface Props {
+    inductions: Induction[];
+}
+
+export const InviterInductions = ({ inductions }: Props) => (
+    <InductionTable.Table
+        columns={INVITER_INDUCTION_COLUMNS}
+        data={getTableData(inductions)}
+        tableHeader="My outstanding invitations"
+    />
+);
+
+const INVITER_INDUCTION_COLUMNS: InductionTable.Column[] = [
+    {
+        key: "invitee",
+        label: "Invitee",
+    },
+    {
+        key: "inviter_voters",
+        label: "Inviter & Voters",
+        className: "hidden md:flex",
+    },
+    {
+        key: "time_remaining",
+        label: "Time remaining",
+        className: "hidden md:flex",
+    },
+    {
+        key: "status",
+        label: "Action/Status",
+        type: InductionTable.DataTypeEnum.Action,
+    },
+];
+
+const getTableData = (inductions: Induction[]): InductionTable.Row[] => {
+    return inductions.map((ind) => {
+        const [allEndorsements] = useFetchedData<any>(
+            getEndorsementsByInductionId,
+            ind.id
+        );
+        const endorsers = allEndorsements
+            ?.map((end: Endorsement): string => end.endorser)
+            .filter((end: string) => end !== ind.inviter)
+            ?.join(", ");
+        const remainingTime = dayjs().to(
+            dayjs(ind.created_at).add(7, "day"),
```

Comment:
this is beautiful... `created_at` is an iso string and dayjs handles that out of the box? amazing!

Reply:
Yeah, pretty cool, right!